Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -32,6 +32,7 @@ public class DafnyProgramVerifier : IProgramVerifier {
       await mutex.WaitAsync(cancellationToken);
       try {
 
+        // IDictionary<ModuleKey, IReadOnlyList<IImplementationTask>>;
```

Comment:
I think this is left over from you showing me how you'd implement Boogie translation caching :)

```suggestion
```